rootless.md

```diff
@@ -0,0 +1,41 @@
+# Shortcomings of Rootless Podman
+
+The following list categorizes the known issues and irregularities with running Podman as a non-root user.  Although currently functional, there is still a number of work items that are under consideration to be added.  These proposed changes are in varying degrees of design and development.
```

Comment:
Sorry for coming late to the game. Would it make sense to add a currency indicator such as `known issues **as of April 2019**`?

Reply:
Actually, we might be able to remove some of these now that rootless single userns landed

Reply:
I would prefer to put it in and start removing them as we go.  @edsantiago I don't think we should do the currency, could be used against us.    Just update it as we find and remove issues.